### ISSUE TITLE
Power On By PCI-E (wake on lan asus)

### ISSUE THREAD

**Issue by BloodRain13** [fbelavenuto/arpl#226]
When the 'Power On By PCI-E' (wake on lan asus) function is enabled in the bios of the ROG STRIX B460-GAMING board
, when I turn off DSM_DS920+_42962, the system shuts down and starts itself again after 2-4 seconds.
If i turn off 'Power On By PCI-E' in the bios, then the system shutdown is normal.
I really need a job WELL please help me figure this out.

**Comment by sisland** [comment]
Hi there

maybe some other device is sending a broadcast message that makes your device Boot Up? It happened to me before and has nothing to do with Loader or anything else.

Please, if you have more LAN devices disconnet all of them while you are testing this and let us know

**Comment by BloodRain13** [comment]
After sending the shutdown command, I took out the LAN cable directly from xpenology. it didn't help, it turned on again. before that, there was a jun dsm 6.2 loader.... there were no such problems. I also tried TrueNAS recently, there shutdown works as expected, but by itself 
The TrueNAS system is very complicated for me.

**Comment by BloodRain13** [comment]
My problem may be related to the fact that I use the original SN and MAC from synology ds 920+?

**Comment by sisland** [comment]
Yes. As far as I know you don't need the SN and MAC from another device to run. Only for quick connect and I highly discourage doing so. (You have tailscale to NOT expose your NAS from internet)

**Comment by chestor37** [comment]
> Yes. As far as I know you don't need the SN and MAC from another device to run. Only for quick connect and I highly discourage doing so. (You have tailscale to NOT expose your NAS from internet)

Why not use real sn and mac for xpenology?
